src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -316,15 +327,22 @@ def open_pr_comment_workflow(pr_id: int) -> None:
     client = installation.get_client()
 
     # CREATING THE COMMENT
-    if not safe_for_comment(gh_client=client, repository=repo, pull_request=pull_request):
+    logger.info("github.open_pr_comment.check_safe_for_comment")
+
+    # fetch the files in the PR and determine if it is safe to comment
+    safe_to_comment, pr_files = safe_for_comment(
+        gh_client=client, repository=repo, pull_request=pull_request
+    )
+
+    if not safe_to_comment:
```

Comment:
If we simply check for truthyness of `pr_files`, we can then determine it's safe to comment and you would not need the additional return value.

What if safe to comment is true but the pr_files is empty?

Reply:
oh yeah i was debating this. this makes sense